ingestion_server/test/integration_tests.py

```diff
@@ -235,7 +235,33 @@ def test04_last_task_in_status_list(self):
         resp_json = resp.json()
         msg = 'There should be one task in the task list now.'
         self.assertEqual(1, len(resp_json), msg)
+    
+    def test05_removed_from_source_not_indexed(self):
+        id_to_check = 10494466  #Index for which we changed manually False to True
+        es = Elasticsearch(
+            host='localhost',
+            port=60001,
+            connection_class=RequestsHttpConnection,
+            timeout=10,
+            max_retries=10,
+            retry_on_timeout=True,
+            http_auth=None,
+            wait_for_status='yellow'
+        )
+        es_query = {
+            "query": {
+                "match_all": {}
```

Comment:
Instead of fetching every single document from Elasticsearch and searching for each ID, you should attempt to retrieve only the single document relevant for this test. There's an example of a similar query in the code already [here](https://github.com/creativecommons/cccatalog-api/blob/master/ingestion_server/ingestion_server/indexer.py#L286) for reference.

Reply:
Done and travis passes as well!